chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -233,45 +359,104 @@ class CurrentUser(
             callback: (Result<Int, Error>) -> Unit
     ) = makeCallback({ syncCurrentUser.sendMessage(roomId, messageText, attachment) }, callback)
 
+    /**
+     * Send a [messageText] to the [roomId]. The [callback] will be called with a [Result] when the
+     * operation is complete. If the operation was successful you will receive an [Int] which
+     * contains the message id, or an [Error] informing you of what went wrong.
+     */
     fun sendSimpleMessage(
             roomId: String,
             messageText: String,
             callback: (Result<Int, Error>) -> Unit
     ) = makeCallback({ syncCurrentUser.sendSimpleMessage(roomId, messageText) }, callback)
 
+    /**
+     * @see [sendSimpleMessage]
+     */
     fun sendSimpleMessage(
             room: Room,
             messageText: String,
             callback: (Result<Int, Error>) -> Unit
     ) = makeCallback({ syncCurrentUser.sendSimpleMessage(room, messageText) }, callback)
 
+    /**
+     * Sends some message [parts] to the [room]. For more information on what the [parts] can be check
+     * out the [docs][https://pusher.com/docs/chatkit/reference/android].
+     * The [callback] will be called with a [Result] when the operation is complete. If the
+     * operation was successful you will receive an [Int] which contains the sent message id, or an
+     * [Error] informing you of what went wrong.
+     *
+     * e.g.
+     * `sendMultipartMessage(room, listOf(`
+     * `NewPart.Inline("hello world"),`
```

Comment:
Is a `type` optional here, defaulting to `text/plain`? If so, we should probably still be explicit so as to suggest that you can specify other types...
Or maybe a `NewPart.Inline("{ \"metadata\": { \"messageHighlight\": \"green\" } }", type = "application/json")` as an additional part would be a nice demonstration.

Reply:
Yeah, the type is default to `text/plain` for Inline only. I can add the type here too so people don't misunderstand :)